### DISCUSSION TITLE
Updating to Foundry 0.8

### DISCUSSION THREAD

**Discussion by rppawlo** [cat-box/aws-foundry-ssl#23] [Q&A] [answered]
Hi,

Thanks for supplying these instructions! Really appreciate the work you put into this!

With the 0.8 series about to be released, it doesn't look like there is an easy way to upgrade automatically. I used these instructions to start a server in Feb 2021 and the node.js version is 12 in my instance. To do an upgrade without reinstall from scratch, node.js must be at least v14. Is there any way you could provide instructions on how to save data, halt the server, start up a copy of foundry 0.8 and import the data from the old server?

Thanks again!

**Comment by cat-box** [comment, marked as answer]
I released a patch in March which addresses the node v14 requirement. You can find it [here](https://github.com/cat-box/aws-foundry-ssl/wiki/Patches#nodejs-v14x-080-compatibility-patch). As for backing up your server, I would use WinSCP/Cyberduck to export the /foundrydata folder

For any future releases/patches, you can find them on the same page in the wiki under "Patches" 🙂

**Reply by rppawlo** [reply]
Thanks!
